packages/cell-toolbar/src/celltoolbartracker.ts

```diff
@@ -509,11 +525,20 @@ export class CellBarExtension implements DocumentRegistry.WidgetExtension {
   }
 
   createNew(panel: NotebookPanel): IDisposable {
-    return new CellToolbarTracker(panel, undefined, this._toolbarFactory);
+    return (this._tracker = new CellToolbarTracker(
+      panel,
+      undefined,
+      this._toolbarFactory
+    ));
+  }
+
+  set visible(value: boolean) {
```

Comment:
Could you add a getter for this?

It is a good practice to define a getter when defining a setter.

Reply:
Getter added.